### ISSUE TITLE
SCDO: failed to parse

### ISSUE THREAD

**Issue by jvendetti** [ncbo/bioportal-project#19]
Received a complaint from a user on the support list that they submitted a new ontology ([Sickle Cell Disease Ontology](http://bioportal.bioontology.org/ontologies/SCDO)), and the status is "Uploaded, Error Rdf".

**Comment by jvendetti** [comment]
User entered a pull URL of https://github.com/scdodev/scdo-ontology/blob/master/data/scdo.owl, which points to an HTML page, instead of the actual ontology file.  The proper URL should be https://raw.githubusercontent.com/scdodev/scdo-ontology/master/data/scdo.owl.

I created a new submission using the proper pull URL.  Verified that the proper ontology source file gets pulled by BioPortal, but the OWL file fails to parse due to a new error.  Investigating...

**Comment by jvendetti** [comment]
Failure occurs when we try to load the data into the triplestore.  Used rapper to try to locate the problem:

```
[ncbo-deployer@ncbo-prd-app-25 6]$ rapper -i rdfxml -o ntriples owlapi.xrdf > data.triples
rapper: Parsing URI file:///srv/ncbo/share/env/production/repository/SCDO/6/owlapi.xrdf with parser rdfxml
rapper: Serializing with serializer ntriples
rapper: Parsing returned 18410 triples
[ncbo-deployer@ncbo-prd-app-25 6]$ rapper -i ntriples -c data.triples
rapper: Parsing URI file:///srv/ncbo/share/env/production/repository/SCDO/6/data.triples with parser ntriples
rapper: Error - URI file:///srv/ncbo/share/env/production/repository/SCDO/6/data.triples:13575 column 214 - Saw ':', expected Graph term <URIref>, _:bnodeID
rapper: Error - URI file:///srv/ncbo/share/env/production/repository/SCDO/6/data.triples:15435 column 131 - Saw ':', expected Graph term <URIref>, _:bnodeID
rapper: Parsing returned 18408 triples
```

It appears that there are invalid values for xml:lang.  I found 7 instances of this in the ontology source file:

- line 2025: <ontology:SCDO_0000340 xml:lang="xref">
- line 3245: <ontology:SCDO_0000484 xml:lang="cost">
- line 5620: <dc:source xml:lang="dc">
- line 12,214: <ontology:SCDO_0000303 xml:lang="the">
- line 20,625: <ontology:SCDO_0000303 xml:lang="http://pediatrics.aappublications.org/content/130/5/926">
- line 23,482: <ontology:SCDO_0000335 xml:lang="dc:source">
- line 24,330: <ontology:SCDO_0000335 xml:lang="adapted">

**Comment by jvendetti** [comment]
I downloaded a copy of this ontology and modified the values of the xml:lang attribute to contain "en".  Created a new submission for this ontology and specified a file upload instead of a pull URL, using the corrected file from my local machine.  Reprocessed the ontology, which succeeded, and deleted 6 previous failed submissions.  Contacted the author to let them know that the ontology is available for browsing.  I also detailed the steps they would need to perform in order to fix the xml:lang tags in their version in GitHub, and gave instructions for entering a correct pull URL if they want to go back to having new versions pulled nightly.
